### PR TITLE
[README] test

### PR DESCRIPTION
**Pull request by maksr51314**
Here is example of pull example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # image_bot
 
-x test changes ..
```
Comment (KnuckVlad) [inline README.md]: test comment

Comment (ilia-yats) [inline README.md]: wtf?!

Comment (GreatSkiv) [inline README.md]: Code is good

File: README.md [hunk 2]
```diff
@@ -1,3 +1,2 @@
 # image_bot
```
Comment (ivanovdn) [inline README.md]: ok

File: README.md [hunk 3]
```diff
@@ -1,3 +1,2 @@
...
```
Comment (igor-vasiurin) [inline README.md]: #right comment

**Pull request merged**